src/App.js

```diff
@@ -6,17 +6,25 @@ import "./app.css";
 const today = new Date();
 
 function App() {
-  const [newTodo, setNewTodo] = useState("Hi! Something will be here");
+  //This state change the list of todos;
+  const [todoList, setTodoList] = useState([]);
+
+  //Callback handler, to ad New Todo to List
+  const addTodo = (newTodo) => {
+    console.log("newTodo", newTodo);
+    setTodoList([...todoList, newTodo]);
+    console.log("newTodo", newTodo);
```

Comment:
You're welcome to remove the console.log statements; they are primarily used for debugging purposes.